docs/en/quick-answers/developing/sgtk-script-authentication.md

```diff
@@ -50,7 +50,7 @@ user = authenticator.get_user()
 
 # print "User is '%s'" % user
 
-# Tells Toolkit which user to use for connecting to Shotgun. Note that this should
+# Tells Toolkit which user to use for connecting to ShotGrid. Note that this should
```

Comment:
Similar to above, these are descriptive comments in example code, so I decided to update them as well.